### PR TITLE
build: add lib flatbuffers

### PR DESCRIPTION
**Pull request by 15927146721**
增加flatbuffers依赖

image-20231020181455682

### CODE REVIEW

**Comment by 15927146721** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by 15927146721** [comment]
I have read the CLA Document and I hereby sign the CLA.

File: flatbuffers/linglong.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package:
+  id: flatbuffers
```
Comment (kamiyadm) [inline flatbuffers/linglong.yaml:2]: name 和 description 补一下

**Comment by kamiyadm** [comment]
你这个截图不太对

**Comment by kamiyadm** [comment]
有空把上面评论的name 和description补充下就可以了

**Review by kamiyadm — APPROVED** [review]

**Pull request merged**